packages/flutter/lib/src/widgets/modal_barrier.dart

```diff
@@ -24,17 +25,20 @@ class ModalBarrier extends StatelessComponent {
   final bool dismissable;
 
   Widget build(BuildContext context) {
-    return new Listener(
-      onPointerDown: (_) {
-        if (dismissable)
-          Navigator.pop(context);
-      },
-      behavior: HitTestBehavior.opaque,
-      child: new ConstrainedBox(
-        constraints: const BoxConstraints.expand(),
-        child: color == null ? null : new DecoratedBox(
-          decoration: new BoxDecoration(
-            backgroundColor: color
+    return new Semantics(
+      container: true,
```

Comment:
This is a bit mysterious to me currently.  What does the container annotation mean and how can I remember when to use it?